mlpf/tfmodel/model_setup.py

```diff
@@ -65,9 +65,9 @@ class ModelOptimizerCheckpoint(tf.keras.callbacks.ModelCheckpoint):
     def on_epoch_end(self, epoch, logs=None):
         super(ModelOptimizerCheckpoint, self).on_epoch_end(epoch, logs=logs)
         weightfile_path = self.opt_path.format(epoch=epoch+1, **logs)
-        try:
+        try:        
```

Comment:
Have we implemented the loading of these optimizer weights somewhere?

Reply:
we use it for restarting/continuing the training, and it's confirmed that it works.

Reply:
https://github.com/jpata/particleflow/blob/june_2022/mlpf/pipeline.py#L248